shared/common-adapters/avatar.render.desktop.js

```diff
@@ -148,6 +148,17 @@ class AvatarRender extends React.PureComponent<Props, State> {
         {this.props.followIconType && (
           <Icon type={this.props.followIconType} style={this.props.followIconStyle} />
         )}
+        {this.props.editable && (
+          <Icon
+            type="iconfont-edit"
+            onClick={this.props.onEditAvatarClick}
+            style={{
+              bottom: this.props.isTeam ? -2 : 0,
+              position: 'absolute',
+              right: this.props.isTeam ? -18 : 0,
+            }}
+          />
```

Comment:
I removed that `onClick` because it's on a layer behind the avatar, and the avatar's `onClick` will take care of it. So, it's as if they're wrapped in a `ClickableBox` but not really.

Reply:
Ah awesome!